NitroxClient/MonoBehaviours/Debugging/NitroxDebug.cs

```diff
@@ -0,0 +1,140 @@
+﻿using System.Collections.Generic;
+using UnityEngine;
+
+namespace NitroxClient.MonoBehaviours.Debugging
+{
+    public class NitroxDebug : MonoBehaviour
+    {
+        public KeyCode EnableDebuggerHotkey;
+        public List<BaseDebugger> Debuggers;
+        private HashSet<BaseDebugger> wereActive;
+        private bool isDebugging;
+        private Rect windowRect;
+
+        public void Awake()
+        {
+            windowRect = new Rect(10, 10, 200, 0);
+
+            EnableDebuggerHotkey = KeyCode.F7;
+            wereActive = new HashSet<BaseDebugger>();
```

Comment:
We prefer setting such fields at declaration, and making them `readonly`. Optionally, you can call `.Clear()` from `Awake`.

```csharp
private readonly HashSet<BaseDebugger> wereActive = new HashSet<BaseDebugger>();
```